airbyte-integrations/connectors/source-hubspot/unit_tests/test_streams.py

```diff
@@ -257,7 +250,7 @@ def test_stream_read_with_legacy_field_transformation(
     stream._sync_mode = SyncMode.full_refresh
 
     if isinstance(stream_class, str):
-        stream_url = stream.retriever.requester.url_base + stream.retriever.requester.path
+        stream_url = stream.retriever.requester.url_base + "/" + stream.retriever.requester.get_path()
```

Comment:
noting that we need to use `get_path()` instead of `path` because `.path` will return raw string which can be an interpolated value `{{ parameters['entity'] }}`